CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ endif(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 
 # Options
 
-option(ENABLE_TESTS "Enable all tests and checks" OFF)
+option(ENABLE_TESTS "Enable all tests and checks" ON)
```

Comment:
@sunweaver: I think we should revert this change - it is inconsistent with everything else in Ayatana. In all our sources we provide defaults for quick and easy builds - testing is optional (mostly for packagers and developers in mind).

Reply:
This was reverted with https://github.com/AyatanaIndicators/ayatana-ido/commit/f19a5f472d282de32ca270afc68a6da8d4edca61.